doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -855,24 +890,6 @@ Modem libraries
       Use the :kconfig:option:`CONFIG_LTE_NETWORK_MODE_LTE_M_NBIOT` or :kconfig:option:`CONFIG_LTE_NETWORK_MODE_LTE_M_NBIOT_GPS` Kconfig option instead.
       In addition, you can control the priority between LTE-M and NB-IoT using the :kconfig:option:`CONFIG_LTE_MODE_PREFERENCE` Kconfig option.
 
-  * Deprecated the :c:macro:`LTE_LC_ON_CFUN` macro.
-    Use the :c:macro:`NRF_MODEM_LIB_ON_CFUN` macro instead.
-
-  * Added a new :c:enumerator:`LTE_LC_EVT_RAI_UPDATE` event that is enabled with the :kconfig:option:`CONFIG_LTE_RAI_REQ` Kconfig option.
-
-  * Updated:
-
-    * To use the :ref:`at_parser_readme` library instead of the :ref:`at_cmd_parser_readme` library.
-    * The :c:func:`lte_lc_neighbor_cell_measurement` function to return an error for invalid GCI count.
-    * The :c:func:`lte_lc_factory_reset` function has been deprecated.
-      Use the ``AT%XFACTORYRESET`` AT command instead.
-      Refer to the :ref:`migration guide <migration_2.8>` for more details.
-    * The :c:enum:`lte_lc_factory_reset_type` type has been deprecated.
-    * The :c:func:`lte_lc_reduced_mobility_get` and :c:func:`lte_lc_reduced_mobility_set` functions have been deprecated.
-      Refer to the :ref:`migration guide <migration_2.8>` for more details.
-    * The :c:enum:`lte_lc_reduced_mobility_mode` type has been deprecated.
-      Refer to the :ref:`migration guide <migration_2.8>` for more details.
-
```

Comment:
Line: 885 :ref:`lib_zzhc` library expands to China Telecom ZZHC library library. So 'library' can be removed.